### PR TITLE
[Issue #3850] Add current and past partners to grants view

### PR DESCRIPTION
**Pull request by rafalkowalski**
##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [x] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [x] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor
screenshot_16

### CODE REVIEW

File: app/grants/views.py [hunk 1]
```diff
@@ -74,14 +74,17 @@ def grants(request):
...
+    now = datetime.datetime.now()
     params = {
         'active': 'grants_landing',
         'title': _('Grants Explorer'),
         'sort': sort,
         'network': network,
         'keyword': keyword,
-        'matchpledges': MatchPledge.objects.filter(active=True).order_by('-amount'),
+        'current_partners': partners.filter(end_date__gte=now).order_by('-amount'),
+        'past_partners': partners.filter(end_date__lte=now).order_by('-amount'),
```
Comment (SaptakS) [inline app/grants/views.py]: Maybe make it `end_date__lt` rather than `end_date__lte`.

**Comment by PixelantDesign** [comment]
Looking good! looks like we just need the robot image at the top and some of those logos in there?
cc: @owocki @willsputra

**Comment by owocki** [comment]
😍

**Comment by danlipert** [comment]
Nice! I don't see any issues

**Comment by willsputra** [comment]
here's the sketch file with the robot illustration
https://github.com/gitcoinco/creative/pull/45/commits/97870f4883e9f8972411ab7d32423e22596e26e7

**Comment by rafalkowalski** [comment]
Current changes:
image

**Comment by rafalkowalski** [comment]
@SaptakS I haven't added any additional migrations, Could you check why new migration file(0016_merge_20190305_1435) is created in travis server?

**Comment by owocki** [comment]
screenshot looks sane to me.. excited for this!

**Pull request merged**